opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/OpenTelemetrySpanBuilderImpl.java

```diff
@@ -49,7 +50,8 @@
 
 class OpenTelemetrySpanBuilderImpl extends SpanBuilder {
   private static final Tracer OTEL_TRACER =
-      GlobalOpenTelemetry.getTracer("io.opentelemetry.opencensusshim");
+      GlobalOpenTelemetry.getTracer(
+          "io.opentelemetry.opencensusshim", OpenCensusLibraryInformation.VERSION);
```

Comment:
That is the OpenCensus API version, not the version of the OC->OTel shim.

You can read the shim version from the `io/opentelemetry/opencensusshim/version.properties` file - it is autogenerated during build for every library in this project. See https://github.com/open-telemetry/opentelemetry-java/blob/696eea790305eaeba0bbb62a23a43dbbfcf29e32/sdk/common/src/main/java/io/opentelemetry/sdk/resources/Resource.java#L112-L122 for another example

Reply:
Ah geez, what was I thinking? You're right. And I hadn't thought to hunt for the properties file: I'm used to generating `BuildInfo` classes that wrap these so it never to me to look. I've updated these.